File: backend/src/main/java/com/codrin/model/Response/PlaylistResponse.java
package com.codrin.model.Response;

import java.util.List;

public class PlaylistResponse {
    private int id;

    private String name;
    private List<TrackResponse> trackList;

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public List<TrackResponse> getTrackList() {
        return trackList;
    }

    public void setTrackList(List<TrackResponse> trackList) {
        this.trackList = trackList;
    }
}

File: backend/Models/Response/PlaylistResponse.cs
namespace backend.Models.Response
{
    public class PlaylistResponse
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public List<TrackResponse> TrackList { get; set; }
        public string AddedBy { get; set; }

        public PlaylistResponse(Playlist playlist)
        {
            Id = playlist.Id;
            Name = playlist.Name;
            AddedBy = playlist.AddedBy;
            TrackList = new List<TrackResponse>();

        }
    }
}
